doc/wakaama_features.rst

```diff
@@ -0,0 +1,213 @@
+Wakaama Features
+----------------
+
+This table shows the LwM2M features that are supported by Wakaama.
+
++----------------------------+----------------------------------------+------------------+-------------------------------+
+| Feature Type               | Feature                                | Wakaama supports | Note                          |
++============================+========================================+==================+===============================+
+| Transport                  | DTLS                                   | v_1.0            | by tinydtls                   |
```

Comment:
I do not understand what the meaning of `v_1.0` is. I assume it is version 1.0 of LwM2M, but it could also be e.g. a release tag of Wakaama.

Reply:
Agree, that's not clear. I'll just change it to 'yes' (and to 'no').